setup.py

```diff
@@ -44,19 +44,19 @@
 
 
 def register_ac():
-    from pathlib import Path
+    from fs import open_fs
     import os
     import re
 
-    home = str(Path.home())
+    home = str(open_fs.home())
     resource_path = 'jina/resources/completions/jina.%s'
     regex = r'#\sJINA_CLI_BEGIN(.*)#\sJINA_CLI_END'
     _check = {'zsh': '.zshrc', 'bash': '.bashrc', 'fish': '.fish'}
 
     def add_ac(k, v):
         v_fp = os.path.join(home, v)
         if os.path.exists(v_fp):
-            with open(v_fp) as fp, open(resource_path % k) as fr:
+            with open('.') as fp, open(resource_path % k) as fr:
```

Comment:
it is designed to write under `~/`, so this change is wrong.

Reply:
However a content type can be specified by rst extension or markdown corresponding to no formatting incurs in code.